client/client.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/hashicorp/errwrap"
 	"github.com/joyent/triton-go/authentication"
+	tt "github.com/joyent/triton-go/errors"
```

Comment:
I don't like `tt` as a package alias for `errors`.  I'd do something more verbose like `tritonErrors` if possible.  I don't like "`terrors`", either, because that becomes confusing to the reader.

Actually, is this awful?

```go
import (
  stderrors "errors"

  "github.com/joyent/triton-go/errors"
)
```

Things like `fmt.Errorf()` are unaffected and it makes it more difficult to introduce a non-standard error.

We could even change `New()` so that calls like:

```
&tt.TritonError{
  StatusCode: response.StatusCode,
￼  Code:       "ResourceNotFound",
}
```

can be replaced by:

```
errors.New(response.StatusCode, "ResourceNotFound")
```

Would you be okay with that?  I'm still mulling the stutter of `tritonError.TritonError` and `StatusCode` + `Code`.  Can those two be folded into the constructor so that `triton-go/errors/New()` derives the `Code` automatically based on the `response.StatusCode`?  Maybe `NewFromStatusCode(response.StatusCode)` does the lookup?